### PR TITLE
Feature/use all markets

### PR DESCRIPTION
**Pull request by bbroeking**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@honey-finance/sdk",
-  "version": "1.1.2",
+  "version": "1.1.3",
```
Comment (BowTiedHeron) [inline package.json:3]: 1.1.3 was already deployed 1 month ago, should this be changed? @bbroeking 
https://www.npmjs.com/package/@honey-finance/sdk?activeTab=versions

File: src/hooks/fetchAllMarkets.ts [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+            if (nft.toString() != '11111111111111111111111111111111') {
+              const totalDebt =
+                reserveInfoList[0].loanNoteExchangeRate
+                  .mul(item.account?.loans[0]?.amount)
+                  .div(new BN(10 ** 15))
+                  .div(new BN(10 ** 6)) //!!
+                  .div(new BN(10 ** 5))
+                  .toNumber() /
+                10 ** 4; //dividing lamport
+              let position: NftPosition = {
```
Comment (bowtiedpyro) [inline src/hooks/fetchAllMarkets.ts:172]: Within the nftposition object would also like to receive the verifiedCreator, is that possible @bbroeking

File: src/hooks/fetchAllMarkets.ts [hunk 3]
```diff
@@ -0,0 +1,201 @@
...
+                obligation: item.publicKey.toString(),
+                debt: totalDebt,
+                nft_mint: new PublicKey(nft),
+                owner: item.account.owner,
+                ltv: 40,
+                is_healthy: getHealthStatus(totalDebt, nftPrice),
```
Comment (bowtiedpyro) [inline src/hooks/fetchAllMarkets.ts:178]: The isHealthy value is not being used anymore @bbroeking and @BowTiedHeron in this way. This still returns the old structure of RISKY, LOW, etc.

**Review by bowtiedpyro — COMMENTED** [review]
Please see comments

File: src/hooks/fetchAllMarkets.ts (hunk 3 above)
Comment (BowTiedHeron) [reply src/hooks/fetchAllMarkets.ts:178]: is_healthy is something else @bowtiedpyro it checks if the NFT should be liquidated or not.
not related to the frontend.

**Review by bowtiedpyro — APPROVED** [review]

**Review by BowTiedHeron — APPROVED** [review]

**Pull request merged**